.gitlab-ci.yml

```diff
@@ -563,10 +613,10 @@ cluster-cleanup:
 ## user cluster cleanup
 user-cluster-cleanup:
   when: always
-  image: harshshekhar15/gitlab-job:v1
+  image: harshshekhar15/gitlab-job:v3
   dependencies:
-    - cluster-setup-check
+    - cluster-create
```

Comment:
`user-cluster-cleanup` job will depend on `cluster-setup-check`

Reply:
This is fixed

Reply:
@harshshekhar15 we have added only 4 stages that are currently passing and we want to merge them in master as a first step. We are going to fix and push the other 4 stages after they are fixed.

Reply:
@nrusinko ok great. Thanks.